docs/Using-Fleet/FAQ.md

```diff
@@ -1,6 +1,6 @@
 # Using Fleet FAQ
 
-- [What do I need to do to switch from Kolide Fleet to FleetDM Fleet?](#waht-do-i-need-to-do-to-switch-from-kolide-fleet-to-fleetdm-fleet)
+- [What do I need to do to switch from Kolide Fleet to new Fleet?](#waht-do-i-need-to-do-to-switch-from-kolide-fleet-to-fleetdm-fleet)
```

Comment:
To migrate to Fleet from Kolide/Fleet, please follow the steps outlined in the [Upgrading Fleet section](http://localhost:2024/docs/deploying/upgrading-fleet) of the documentation.

Reply:
Question becomes:

How do I switch to Fleet from Kolide Fleet?

Answer (from line 36 onwards) becomes what you have commented above.

Reply:
Hey @mike-j-thomas, I caught that the section was repeating itself. I suggest we not update the text on line 40 but delete it entirely and include the link to the other doc in the first sentence. I also searched the document for any other references to Kolide to make sure I didn't miss anything.